src/main/java/uk/gov/companieshouse/mapper/DissolutionDirectorResponseMapper.java

```diff
@@ -0,0 +1,20 @@
+package uk.gov.companieshouse.mapper;
+
+import org.springframework.stereotype.Service;
+import uk.gov.companieshouse.model.db.dissolution.Dissolution;
+import uk.gov.companieshouse.model.dto.dissolution.DissolutionDirectorPatchResponse;
+
+@Service
+public class DissolutionDirectorResponseMapper extends ResponseMapper {
+
+    public DissolutionDirectorPatchResponse mapToDissolutionDirectorPatchResponse(Dissolution dissolution) {
+        final DissolutionDirectorPatchResponse response = new DissolutionDirectorPatchResponse();
+
+        response.setLinks(generateLinks(
```

Comment:
not a requested change but one that you should keep in mind is the concept of composition over inheritance. Imagine ResponseMapper being injectable rather than extendable. Therefore you could mock out the ResponseMapper, whereas with inheritance you have to unit test the same implementation (inherited code) in multiple places.